samples/cors/ts/server/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "esModuleInterop": true,
+    "target": "ES2020",
```

Comment:
Dang. You're not messing around!

Reply:
Oh... I really don't know what I'm doing when it comes to targeting different versions to be honest - do you have a recommendation for a reasonable default?

Reply:
I think (at most) we typically target ES2015 (and honestly I'm not sure if there's any reason to obey that restriction here):

https://github.com/Azure/azure-sdk-for-js/blob/94edf25d2ebe4b456d416b4e087500c57724dcbb/tsconfig.json#L3